internal/db/schema/migrations/oss/postgres/25/01_static_host_view.up.sql

```diff
@@ -0,0 +1,22 @@
+begin;
+
+create view static_host_with_set_memberships as
```

Comment:
Please add a comment to this view explaining its purpose and details regarding the unique return value of the `set_ids` column (the delimitation and that there is no size limits currently).